util/personal_names.py

```diff
@@ -1,14 +1,36 @@
+from fuzzywuzzy import fuzz
+from nameparser import HumanName
+from nose.tools import set_trace
+import re
+
+from permanent_work_id import WorkIDCalculator;
+
+
 """Fallback algorithms for dealing with personal names when VIAF fails us."""
 
 def is_corporate_name(display_name):
     """Does this display name look like a corporate name?"""
+    corporations = ['National Geographic', 'Smithsonian Institution', 
+        'Verlag', 'College', 'University',  
+        'Harper & Brothers', 'Harper &amp; Brothers', 'Williams & Wilkins', 'Williams &amp; Wilkins', 
```

Comment:
Rather than listing these twice it might be better to perform entity substitution on the incoming display name.